src/renderer/cursor_renderer/mod.rs

```diff
@@ -22,6 +22,10 @@ pub struct CursorSettings {
     antialiasing: bool,
     animation_length: f32,
     animate_in_insert_mode: bool,
+    springloaded: bool,
```

Comment:
Would it make sense to have a cursor_trail enum similar to the VfxMode rather than a boolean here? I feel like that way it leaves the door  open for future trails.

Alternatively I could see an argument that this is just a better implementation of the trail animation... I don't really know without playing with it a bit more.

Reply:
That's a good idea. I'll have a look into it in the next couple of days.